slices/dbus-system-bus-common.yaml

```diff
@@ -0,0 +1,23 @@
+package: dbus-system-bus-common
+
+essential:
+  - dbus-system-bus-common_copyright
+
+slices:
+  # Depends on adduser, however it's used by the dbus postinst
+  # to create the following user:
+  # 'messagebus' with home '/var/run/dbus' and disabled password
```

Comment:
I thought the user did not have any home dirs:

```sh
    in_sysroot adduser --system \
            --quiet \
            --home /nonexistent \                                                                                       
            --no-create-home \
            --disabled-password \
            --group "$MESSAGEUSER"
```

Reply:
I was fooled by the `MESSAGEHOME` variable that seems unused :-) corrected it, thanks!